catkin_tools/config.py

```diff
@@ -35,6 +35,7 @@
 install: config --install
 p: create pkg
 test: build --verbose --make-args test --
+roslint: build -iv --make-args roslint --
```

Comment:
Yes, for example I have 5 errors and I get:

```
[rviz_visual_tools] Total errors found: 5                                                                                                                                                                                                      
[rviz_visual_tools] make[3]: *** [roslint_rviz_visual_tools] Error 1                                                                                                                                                                           
[rviz_visual_tools] make[2]: *** [CMakeFiles/roslint_rviz_visual_tools.dir/all] Error 2                                                                                                                                                        
[rviz_visual_tools] make[1]: *** [CMakeFiles/roslint.dir/rule] Error 2                                                                                                                                                                         
[rviz_visual_tools] make: *** [roslint] Error 2                                                                                                                                                                                                
[rviz_visual_tools] <== '/home/dave/ros/current/ws_ros/build/rviz_visual_tools/build_env.sh /usr/bin/make roslint --jobserver-fds=3,5 -j' failed with return code '2'                                                                         
Failed   <== rviz_visual_tools [ 3.3 seconds ]                                                                                                                                                                                                 
[build] There were '1' errors:                                                                                                                                                                                                                 

Failed to build package 'rviz_visual_tools' because the following command:

# Command to reproduce:
cd /home/dave/ros/current/ws_ros/build/rviz_visual_tools && /home/dave/ros/current/ws_ros/build/rviz_visual_tools/build_env.sh /usr/bin/make roslint --jobserver-fds=3,5 -j; cd -

# Path to log:
cat /home/dave/ros/current/ws_ros/build/build_logs/rviz_visual_tools.log

Exited with return code: 2 
                                                                                                                                                                                                                                               
[build] Runtime: 3.4 seconds 
```